### ISSUE TITLE
Ring variable in app.pcss file is defined as a function

### ISSUE THREAD

**Issue by jesi-rgb** [huntabyte/shadcn-svelte#883]
### Describe the bug

The docs very conveniently explain that colors in the pcss file for the themes must be defined without the color space function.

The automatic installer, however, does include the --ring variable as a function in the `.dark` class selector:
```diff
  .dark {
    --background: 192.2 4% 10.9%;
    --foreground: 210 40% 98%;

    --muted: 217.2 4% 17.5%;
    --muted-foreground: 215 4% 65.1%;

    --popover: 192.2 4% 4.9%;
    --popover-foreground: 192.2 4% 98%;

    --card: 192.2 84% 4.9%;
    --card-foreground: 210 40% 98%;

    --border: 192.2 4% 17.5%;
    --input: 192.2 4% 17.5%;

    --primary: 192.2 4% 21%;
    --primary-foreground: 192.2 4% 90.2%;

    --secondary: 217.2 32.6% 17.5%;
    --secondary-foreground: 210 40% 98%;

    --accent: 192.2 9% 17.5%;
    --accent-foreground: 192.2 4% 98%;

    --destructive: 0 62.8% 30.6%;
    --destructive-foreground: 210 40% 98%;

+   --ring: hsl(212.7, 26.8%, 83.9);
  }
}
```

the bug can easily be solved removing the function and parentheses, but it was confusing since none of the ring behaviours I expected (focusing on textarea, for example) worked.

Very small but important!

### Reproduction

Install the latest svelte-shadcn version as usual

### System Info

```bash
System:
    OS: macOS 14.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 1.71 GB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.5.0 - /opt/homebrew/bin/node
    Yarn: 4.1.0 - /opt/homebrew/bin/yarn
    npm: 10.2.4 - /opt/homebrew/bin/npm
    pnpm: 8.6.12 - ~/Library/pnpm/pnpm
    bun: 1.0.0 - ~/.bun/bin/bun
  Browsers:
    Chrome: 119.0.6045.159
    Safari: 17.1
  npmPackages:
    @sveltejs/kit: ^2.0.0 => 2.5.3 
    bits-ui: ^0.19.5 => 0.19.5 
    mode-watcher: ^0.2.2 => 0.2.2 
    svelte: ^4.2.7 => 4.2.12
```

### Severity

annoyance

**Comment by shyakadavis** [comment]
Hi, everyone;

I think I narrowed down the area of flaw;

This issue has never been the case for me, and even just now, did a fresh install, and the ring variable in the `.dark` section is correct, i.e `--ring: 0 0% 83.1%;`

I suspected the error happens when choosing a `base color`, since in my case, `Neutral` this works as expected.

This issue only occurs when one chose `Slate` as their base colour (I'm guessing this was your case, @jesi-rgb )

---

Will keep digging around in the registry functions for a fix. 🙂

**Comment by jesi-rgb** [comment]
Oh wow, @shyakadavis outstanding! That's in fact true, I chose Slate, just to see how it looked.

**Comment by shyakadavis** [comment]
Also, allow me to add to the issues on the CLI level;

I don't know how I forgot to bring this to your attention, (thanks for opening this, Jesi) but there are other styles not included, but the docs explicitly state that they are;

i.e `font-feature-settings: "rlig" 1, "calt" 1;`

I always find myself manually adding these two, alongside the `antialiased` class, to the body.

---
For consideration if this unintended. 🙂

**Comment by huntabyte** [comment]
Hey @shyakadavis, I think it's fine if we include that! Though with the latest release using `Geist` as the Font, shad has turned those font feature settings off!

**Comment by huntabyte** [comment]
This issue lies somewhere within the `build-registry` script. The fact that it only applies to slate is a bit odd, perhaps there is a typo that isn't immediately obvious somewhere.

**Comment by HubbeDev** [comment]
@huntabyte In [static/registry/themes/slate.json](https://github.com/huntabyte/shadcn-svelte/blob/e48bd234c64aa2d41871981f52a520d34a373a3a/apps/www/static/registry/themes/slate.json#L45) the color is specified with HLS. Is this file  autogenerated from somewhere else? Just took a quick look but what i could tell this file is manually made and is used to create the css file right?

**Comment by huntabyte** [comment]
That would be correct @HubbeDev !

**Comment by huntabyte** [comment]
Closed by #922

**Comment by kjmj** [comment]
Folks, I think this is back. I've verified this as happening today. Here ie my `components.json`

```
{
	"$schema": "https://shadcn-svelte.com/schema.json",
	"style": "new-york",
	"tailwind": {
		"config": "tailwind.config.js",
		"css": "src/app.css",
		"baseColor": "slate"
	},
	"aliases": {
		"components": "$lib/components",
		"utils": "$lib/utils/shadcn-utils"
	},
	"typescript": true
}

```
